src/core/lib/struct.jsx

```diff
@@ -78,4 +78,21 @@ const pick = (sourceObject, keysToPick) => {
 const typeMatch = (input) =>
   input !== null && typeof input === "object" && !Array.isArray(input);
 
-return { deepFieldUpdate, isEqual, pick, toOrdered, typeMatch };
+/**
```

Comment:
I think this could be moved to core/lib/stringUtils and renamed to "normalize"

change label to just value

Reply:
Since it's not related to Structs

Reply:
Otherwise looks great, exactly what I was asking, ty

Reply:
okay I will update it